salt/states/git.py

```diff
@@ -610,6 +610,10 @@ def latest(
     """
     ret = {"name": name, "result": True, "comment": "", "changes": {}}
 
+    # If we call from a runner, __env__ will not be populated, so we need to do so.
+    if "saltenv" in kwargs:
+        __env__ = kwargs.pop("saltenv")
```

Comment:
If it's run as a state, from what I understand, it would have `__env__` packed in.  I want to say we only want to use this if it's run directly as a function like it is in the winrepo runner, but I may be wrong.  Can investigate more.

Reply:
Went a different route with the solution, resolving this.